app/townsquare/management/commands/calculate_activity_scores.py

```diff
@@ -0,0 +1,16 @@
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from dashboard.models import Activity
+from townsquare.views import activity_score
+
+class Command(BaseCommand):
+
+        def handle(self, *args, **options):
+            recent_activities = Activity.objects.filter(created__gt=timezone.now() - timezone.timedelta(hours=72))
```

Comment:
In [5]: Activity.objects.filter(created__gt=timezone.now() - timezone.timedelta(hours=72)).count()
Out[5]: 2149

about 2k scores to calculate every 72 h.. (for now)... well probably see a 10x increase during next grants round